great-idea-website/css/index.css

```diff
@@ -63,4 +63,89 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+ body{
+	 padding:10px;
+ }
+
+ header nav{
+	 display:inline-block;
+	 padding: 10px 100px 0px 110px;
+	 /* border:1px solid red; */
+ }
+ header nav a{
+	 color:gray;
+	 margin: 20px;
+	 text-decoration:none;
+	 display:inline-block;
+
+ }
+
+ header img.logo{
+
+	display: inline-block;
+	margin-left:130px;
+
+ } 
```

Comment:
I'd suggest adding a `vertical-align` property here set to `middle`. Makes the text look a bit more even. 🙂 

Also worth noting: img tags are by default inline elements. `nav` elements however behave as block by default (same as divs). You can view this in the chrome dev tools if you look under computed properties.
![image](https://user-images.githubusercontent.com/8473052/56940345-cbbc2d00-6adb-11e9-9348-74066918b0e0.png)